app/views/tickets/show.html.haml

```diff
@@ -0,0 +1,57 @@
+%script{:src => "//maps.google.com/maps/api/js?key=#{ENV['MAPS']}"}
+%script{:src => "//cdn.rawgit.com/mahnunchik/markerclustererplus/master/dist/markerclusterer.min.js"}
+%script{:src => "//cdn.rawgit.com/printercu/google-maps-utility-library-v3-read-only/master/infobox/src/infobox_packed.js", :type => "text/javascript"}
```

Comment:
???
If you want to enable external script - load it in `<head>` section in `application.html`
Anyway you do not need anything else but maps.google.com